src/pobox.erl

```diff
@@ -198,21 +310,38 @@ notify(cast, _Msg, _State) ->
     keep_state_and_data;
 notify({call, From}, Msg, Data) ->    
     handle_call(From, Msg, notify, Data);
-notify(info, Msg, Data) ->    
+notify(info, Msg, Data) ->
     handle_info(Msg, notify, Data).
         
 
 %% @private
-handle_call(From, {resize, NewSize}, _StateName, S=#state{buf=Buf}) ->
-    {keep_state, S#state{buf=resize_buf(NewSize,Buf)}, 
-        [{reply, From, ok}]};
 handle_call(From, {post, Msg}, StateName, S=#state{buf=#buf{max=Size, size=Size}}) ->
     gen_server:reply(From, full),
     ?MODULE:StateName(cast, {post, Msg}, S);
 handle_call(From, {post, Msg}, StateName, S) ->
     gen_server:reply(From, ok),
     ?MODULE:StateName(cast, {post, Msg}, S);
-
+handle_call(From, {resize, NewSize}, _StateName, S=#state{buf=Buf}) ->
```

Comment:
Might be useful to have a `size()` call. However, I worry that people would use it for load balancing potentially causing memory issues in heavily loaded systems.

Reply:
I guess I meant current usage rather than buffer size.

Reply:
Take a look at `usage` function. See if that works for you.